mesa/agent.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, unique_id: int, model: "Model") -> None:
         self.unique_id = unique_id
         self.model = model
         self.pos: Optional[Position] = None
+        self.heading = 90
```

Comment:
I think we should define a new class for the new attribute and methods, `AgentWithHeading` / `AgentWithDirection` / if you have a better name for it. Because not all models need the feature.